backend/prompt_studio/prompt_studio_output_manager/serializers.py

```diff
@@ -25,4 +26,33 @@ def to_representation(self, instance):
             )
             token_usage = {}
         data["token_usage"] = token_usage
+        # Get the coverage for the current tool_id and profile_manager_id
+        try:
+            # Fetch all relevant outputs for the current tool and profile
+            related_outputs = (
```

Comment:
NIT: Instead of `output` / `outputs` let's rename it to `prompt` / `prompts` to improve the understanding - let's try to correct this variable name eventually. So you could rename this as `related_prompts` and similarly iterate with `for prompt in prompts`

Reply:
I don't understand why we need to rename this as `prompts` since these data are outputs of each prompt. renaming it as prompt will be more confusing instead `prompt_output` will be more appropriate.

Reply:
I mentioned `prompt` extending from the `Prompt` based model we have and the use of it within the prompt service. `prompt_output` probably works better as you mentioned. We'll worry about this readability later